tools/eslint-fuzzer.js

```diff
@@ -10,13 +10,26 @@
 //------------------------------------------------------------------------------
 
 const assert = require("assert");
-const lodash = require("lodash");
+const mapValues = require("map-values");
```

Comment:
I don't think this is a useful dependency. Now it can be replicated with:

```js
Object.fromPairs(Object.entries(obj).map(([value, key]) => cb(value, key, obj))))
```

Or likely something easier to read and more specific to the situation

Reply:
I would prefer to use the dependency. I think

```js
mapValues(ruleConfigs, sample)
```

is way more readable than

```js
Object.fromEntries(Object.entries(ruleConfigs).map(([value, key]) => [value, sample(key)]))
```

In my opinion, the addition of a tiny (16K) module is worth the readability here.

Reply:
Why use a “tiny 16k dependency” when you can use a 1-liner in some other file? It makes zero sense for a PR that attempts to reduce dependencies. 

Like I said, that’s the exact equivalent but you can find better solutions for the 2 places where it’s used.

Reply:
I hadn't thought of creating a function. That's a good idea. I'll do that and remove the dependency.

Reply:
Object.fromEntries is not available until Node 12. I have added a TODO comment to replace this dependency with a helper function when we drop support for Node 10.

Resolved in a28cf2db1.